### PR TITLE
Feature/39 buttons add and remove images and labels from dataset

### LINKED ISSUE #39: feature/add/remove images and add/remove labels from a dataset

### PR DESCRIPTION
**Pull request by NicolasVial**
This PR will : 

- Add a button to be able to remove an image from a dataset (in DisplayImageActivity).
- Add a menu in DisplayDatasetActivity which contains a button to be able to modify the categories.
- Can modify the name of the dataset and see all the pictures of a certain category.

Comments : 

- All the buttons won't actually work for the moment (Only the UI). This will be added when switching from the dummyDataset to the real dataset management model (see #50), probably next week.

Still to be done in this PR:
- Add somme little tests

### CODE REVIEW

**Review by Nitwix — APPROVED** [review]
Overall good, some details noted.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -20,4 +20,8 @@
     <string name="learning_cat_0_descr">red</string>
     <string name="learning_cat_1_descr">green</string>
     <string name="learning_cat_2_descr">blue</string>
+    <string name="modify_categories">Modify categories</string>
+    <string name="dataset_name">Dataset Name</string>
+    <string name="imagetobedisplayed">ImageToBeDisplayed</string>
```
Comment (bbjubjub2494) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="imagetobedisplayed">image to be displayed</string>
```

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -20,4 +20,8 @@
...
+    <string name="buttontoremovepicturefromdataset">buttontoremovepicturefromdataset</string>
```
Comment (bbjubjub2494) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="buttontoremovepicturefromdataset">button to remove picture from dataset</string>
```

**Review by bbjubjub2494 — CHANGES_REQUESTED** [review]
just 3 nitpicks wrt string resources

**Review by bbjubjub2494 — APPROVED** [review]
My comments have been addressed

**Pull request merged**